src/index.js

```diff
@@ -24,6 +24,9 @@ const sugar = {
   }
 }
 
+$.rAF = typeof requestAnimationFrame !== 'undefined' ? requestAnimationFrame : ((callback) => callback())
+$.rIC = typeof requestIdleCallback !== 'undefined' ? requestIdleCallback : ((callback) => callback())
```

Comment:
follow-up to:
> lets keep the type check (thx for checking out options) and instead of do nothing log `console.error('requestAnimationFrame no found')` or something? make it resilient and informative in case it doesnt exist. thoughts?

at one point, i've got it falling back to `((callback) => callback())`, but on second thought that's a risky fallback, cuz you risk infinite recursion, since `requestAnimationFrame` is typically called recursively